frontend/public/components/monitoring.tsx

```diff
@@ -470,7 +470,7 @@ const AlertMessage: React.FC<AlertMessageProps> = ({ alertText, labels, template
   return <p>{messageParts}</p>;
 };
 
-const HeaderAlertMessage: React.FC<{ alert: Alert; rule: Rule }> = ({ alert, rule }) => {
+export const HeaderAlertMessage: React.FC<{ alert: Alert; rule: Rule }> = ({ alert, rule }) => {
```

Comment:
This is being exported, but it looks like you are not actually using this?

Reply:
removed export.